provider/doc_edits.go

```diff
@@ -109,6 +109,26 @@ func targetedSimpleReplace(filePath, from, to string) tfbridge.DocsEdit {
 	}
 }
 
+func targetedReplace(filePath, from, to string) tfbridge.DocsEdit {
+	fromBytes, toBytes := []byte(from), []byte(to)
+	return tfbridge.DocsEdit{
+		Path: filePath,
+		Edit: func(_ string, content []byte) ([]byte, error) {
+			if bytes.Contains(content, fromBytes) {
```

Comment:
Unfortunately this does not ensure that toBytes does not digress, and yet we copied it from the source material. IN this case the toBytes is the bit that contains the notes we're copying.

Reply:
You're totally right. The new check is more specific to the situation and checks that both the added text and the fromBytes have not changed, else it errors.